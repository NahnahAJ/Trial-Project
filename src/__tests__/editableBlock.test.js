import { render } from '@testing-library/react';
import EditablePage from '../components/editablePage';
import getCaretCoordinates from '../components/getCoordinates';

test('renders the EditableBlock component snapshot', () => {
  const { asFragment } = render(
      <EditablePage />
  );
  expect(asFragment()).toMatchSnapshot();
});

test('renders the EditableBlock component snapshot', () => {
  const { asFragment } = render(
      <getCaretCoordinates />
  );
  expect(asFragment()).toMatchSnapshot();
});

