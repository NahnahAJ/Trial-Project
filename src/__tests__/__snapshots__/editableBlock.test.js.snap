// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the EditableBlock component snapshot 1`] = `
<DocumentFragment>
  <div
    class="Page"
  >
    <p
      class="Block"
      contenteditable="true"
    />
  </div>
</DocumentFragment>
`;

exports[`renders the EditableBlock component snapshot 2`] = `
<DocumentFragment>
  <getcaretcoordinates />
</DocumentFragment>
`;
